OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -508,8 +508,8 @@ public void Tick(PlayerResources pr)
 
 			if (pm.PowerState != PowerState.Normal)
 			{
-				if (--Slowdown <= 0)
-					Slowdown = Queue.Info.LowPowerSlowdown;
+				if ((Slowdown -= 100) <= 0)
```

Comment:
This looks weird, why didn't you use just `if (Slowdown - 100 <= 0)`?

Reply:
Sorry, i haven't answered, because i wasn't sure. But looks like `-=` is correct. Slowdown needs to tick down, otherwise unit won't be built.

Reply:
It would be cleaner to split this onto two lines:
```csharp
Slowdown -= 100;
if (SlowDown < 0)
    ...
```